src/lib/db/migrations/20230310111029_initial.ts

```diff
@@ -40,7 +40,9 @@ export async function up(knex: Knex): Promise<void> {
     def.integer('original_token_id').defaultTo(null)
     def.datetime('created_at').notNullable().defaultTo(now())
     def.datetime('updated_at').notNullable().defaultTo(now())
+    def.uuid('reason').notNullable()
```

Comment:
this will need to be nullable as we only have a revocation reason if we have revoked a cert

Reply:
yea, this has been updated along with some other little changes on friday, did not got to the e2e